src/types.ts

```diff
@@ -1,69 +1,66 @@
 import { SigningCosmWasmClient } from "@cosmjs/cosmwasm-stargate"
 import { OfflineSigner } from "@cosmjs/proto-signing"
-import { Keplr } from "@keplr-wallet/types"
+import { SigningStargateClient } from "@cosmjs/stargate"
+import { ChainInfo, Keplr } from "@keplr-wallet/types"
 import WalletConnect from "@walletconnect/client"
 
 import { KeplrWalletConnectV1 } from "./connectors"
 
 export type WalletClient = Keplr | KeplrWalletConnectV1
 
+export enum WalletType {
+  Keplr = "keplr",
+  WalletConnectKeplr = "wallet_connect_keplr",
```

Comment:
kinda feel like `wallet_connect_keplr` could be `walletconnect_keplr` but doesnt matter haha 🤷

Reply:
so true <3